nx-X11/programs/Xserver/hw/nxagent/compext/Compext.c

```diff
@@ -4523,7 +4523,7 @@ static Bool _NXCollectInputFocusHandler(Display *dpy, xReply *rep, char *buf,
 
   DeqAsyncHandler(dpy, state -> handler);
 
-  Xfree(state -> handler);
+  free(state -> handler);
 
   state -> handler = NULL;
```

Comment:
I have introduced the FREE_SAFE macro to nxcomp. I suggest using it here, too. Would safe many lines...